services/order.py

```diff
@@ -0,0 +1,45 @@
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from django.utils.datetime_safe import datetime
+
+from db.models import Order, User, Ticket
+
+
+def create_order(tickets: list, username: str, date: datetime = None) -> Order:
+    if not tickets:
+        raise Exception("No tickets to create")

```

Comment:
Raising a generic `Exception` is not a best practice. Consider raising a more specific exception, such as `ValueError`, to provide clearer context about the error.